mlir/test/Transforms/scf-if-utils.mlir

```diff
@@ -73,3 +73,15 @@ func.func @outline_empty_if_else(%cond: i1, %a: index, %b: memref<?xf32>, %c: i8
   }
   return
 }
+
+// -----
+
+// This test checks scf utils can work on llvm func.
```

Comment:
From the CHECK lines alone, it is unclear whether this actually exercises the code that was changed: outlining. It may well be doing nothing at all and printing the function.